Source/GameEngine/Models/Game.cs

```diff
@@ -2,87 +2,69 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
+using System.Threading;
 
 namespace GameEngine.Library.Models
 {
     public class Game
     {
-        public int GameID { get; set; }
-        public List<User> Users { get; set; }
-        public int GameBoardID { get; set; }
-        public GameBoard Squares { get; set; }
+        private Die dice;
```

Comment:
Die or dice???

Reply:
It should be die, we have now changed it, thanks! :)